### ISSUE TITLE
Stats improvements

### ISSUE THREAD

**Issue by mrzor** [tinyproxy/tinyproxy#288]
Greetings!

There are a couple of things that would make `tinyproxy.stats` more actionable for me :

1. Number of requests should ignore stats requests or count them separately.
2. Time elapsed since last (non stat) request (or alternatively, timestamp of last (non stat) request)

Both can be implemented in practice by doing some form of log processing. 
This doesn't feel like a complicated thing to add - but I haven't checked the source to have an educated opinion on this. 

Sorry I'm not attaching a PR to this. I thought I'd still share the idea - maybe some other people have similar ideas to stats improvements that would make their life easier.

Dear rofl0r, interested to have your thoughts on this. Thanks for all the fish!

**Comment by rofl0r** [comment]
i agree that the stats feature is pretty bare at the moment, to the point of being almost useless, and could use some polish. (optimally with some mock screenshots prior to working on a PR).

>    Number of requests should ignore stats requests or count them separately.

ack

>    Time elapsed since last (non stat) request (or alternatively, timestamp of last (non stat) request)

mhm, this could make sense to give a quick impression about whether the proxy is being used constantly.
